storage/simulation/cluster.go

```diff
@@ -97,6 +100,12 @@ func (c *Cluster) addStore(nodeID proto.NodeID) *Store {
 	s := n.addNewStore()
 	storeID, _ := s.getIDs()
 	c.stores[storeID] = s
+
+	// Save a sorted array of store IDs to avoid having to calculate them
+	// multiple times.
+	c.storeIDs = append(c.storeIDs, storeID)
+	sort.Sort(proto.StoreIDSlice(c.storeIDs))
```

Comment:
Indeed.  But we're only going to have under 10,000 usually, so I don't think it's going to be the bottleneck here.